docs/static/chart/opensource-config.yaml

```diff
@@ -96,6 +96,34 @@ dragonchain:
         requests:
           storage: 2Gi
 
+
+  # -- dragonchain.verificationNotifications --
+  # This Level 1 Dragonchain configuration will send received verifications via HTTP to up to 10 unique URL's per level configured here.
```

Comment:
Any reason for the limitation of 10? If the user wants to send to 1000 endpoints, and it slows down their chain, shouldn't that be on them?

Reply:
Good catch. This limitation was decided against. removed